MapboxNavigation/RouteManeuverViewController.swift

```diff
@@ -132,7 +132,9 @@ class RouteManeuverViewController: UIViewController {
             destinationLabel.unabridgedText = routeProgress.currentLeg.destination.name ?? routeStepFormatter.string(for: routeStepFormatter.string(for: routeProgress.currentLegProgress.upComingStep, legIndex: routeProgress.legIndex, numberOfLegs: routeProgress.route.legs.count, markUpWithSSML: false))
         } else if let upComingStep = routeProgress.currentLegProgress?.upComingStep {
             updateStreetNameForStep()
-            showLaneView(step: upComingStep)
+            if routeProgress.currentLegProgress.alertUserLevel == .medium || routeProgress.currentLegProgress.alertUserLevel == .high {
```

Comment:
We also call `showLaneView(step:)` in `RoutePageViewControllerDelegate.routePageViewController(_:willTransitionTo:)`. Wouldn't this mean that if you swipe to the current step, you'll see the lane view, but when you get an update from `RouteManeuverViewController.notifyDidChange(routeProgress:secondsRemaining:)`, the lane view disappears (for the same step at a low alert level)?

Can we move the show and hide logic to `showLaneView(step:)` and perhaps take routeProgress or alertLevel as a parameter?